x-pack/plugin/monitoring/src/main/java/org/elasticsearch/xpack/monitoring/MonitoringTemplateRegistry.java

```diff
@@ -78,7 +78,7 @@
      * writes monitoring data in ECS format as of 8.0. These templates define the ECS schema as well as alias fields for the old monitoring
      * mappings that point to the corresponding ECS fields.
      */
-    public static final int STACK_MONITORING_REGISTRY_VERSION = Version.V_8_0_0.id + 1;
+    public static final int STACK_MONITORING_REGISTRY_VERSION = Version.V_8_2_0.id + 1;
```

Comment:
For now while we discuss how we want to handle these registry version numbers, this should increment the 8.0 version number instead of increasing the version number to 8.2

```suggestion
    public static final int STACK_MONITORING_REGISTRY_VERSION = Version.V_8_0_0.id + 2;
```

Reply:
More information on version weirdness: https://github.com/elastic/elasticsearch/issues/84147 (standardizing) and https://github.com/elastic/elasticsearch/issues/84149 (the prerequisite to standardizing)